packages/core/src/env.ts

```diff
@@ -1,16 +1,15 @@
 import z from 'zod'
+import { createEnv } from "@t3-oss/env-core"
 
 import '@latitude-data/env'
 
-const envvars = z.object({
```

Comment:
Doing this breaks production build because it tries to find `DATABASE_URL` at build time. With t3-env this is easy fixable by using `skipValidation` with `BUILDING_CONTAINER` flag as we did in web.

Reply:
why? I don't get it why is this code evaluated at build time?

Reply:
Because it's imported by nextjs bundler. And because is imported zod `parse` function is run

Reply:
next bundler runs the code of its dependencies at compile time? :drunk: